src/main-todo/resources/category/java/common.xml

```diff
@@ -268,105 +268,6 @@ public class CalendarStuff {
     </rule-->
 
 
-    <rule name="OptimizeMapOrSetForEnum"
```

Comment:
OptimizeMapOrSetForEnum is also removed from common.xml: correct?

Reply:
yes, from the todo-common